syft/license/license_test.go

```diff
@@ -20,6 +20,11 @@ func TestParseExpression(t *testing.T) {
 			expression: "MIT OR Apache-2.0",
 			want:       "MIT OR Apache-2.0",
 		},
+		{
+			name:       "accept redundant parentheses",
```

Comment:
This test case was never failing on main, but it was the first place I looked, so adding it seems worthwhile to me.